src/pages/homePage/HomePage.tsx

```diff
@@ -1,12 +1,117 @@
 import './HomePage.scss';
 
+import React, { useContext, useEffect, useState } from 'react';
+
+import { HomeCard } from './components/HomeCard';
 import { Image } from '@chakra-ui/react';
-// import { Link } from 'react-router-dom';
-import React from 'react';
+import { RadarContext } from 'navigation/context';
 import { aboutContentList } from 'pages/about/AboutContent';
 import logo from '../../assets/FTRDRR.svg';
+import { supabase } from 'helpers/databaseClient';
+
+const VERSION = process.env.REACT_APP_DISASTER_DATA_VERSION || 'version- 01';
 
 export const HomePage: React.FC = () => {
+  const [projectsToUse, setProjectsToUse] = useState<any>([]);
+  const [TechprojectsToUse, setTechProjectsToUse] = useState<any>([]);
+  const [DisasterprojectsToUse, setDisasterProjectsToUse] = useState<any>([]);
+  const { projectsGroup } = useContext(RadarContext);
+  const MAX_PROJECTS = 3;
+
+  // projects
+  useEffect(() => {
+    if (projectsGroup.length) {
+      const firstThreeProjects = projectsGroup.slice(0, MAX_PROJECTS);
+      setProjectsToUse(firstThreeProjects);
+    } else {
+      void getProjects();
+    }
+  });
+
+  // tech
+  useEffect(() => {
+    if (projectsGroup.length) {
+      const techprojects = projectsGroup;
+      setTechProjectsToUse(techprojects);
+    } else {
+      void getTechProjects();
+    }
+  });
+
+  // disaster
+  useEffect(() => {
```

Comment:
We can combine all `useEffects` into one with the `[]` dependancy so that the data collection happens only once on initial component mounting